src/django/api/templates/mail/api_limit_admin_subject.txt

```diff
@@ -0,0 +1 @@
+Open Apparel Registry contributor API limit notice
```

Comment:
Consider changing this admin-facing email title to include the contributor name so that mail clients don't "helpfully" stack up notifications for different contributors.

One possibility
```
OAR API limit exceeded - {{contributor_name}}
```